### DISCUSSION TITLE
[Q&A] 请教自部署access code的问题

### DISCUSSION THREAD

**Discussion by xrellana** [lobehub/lobehub#937] [General Question] [answered]
-e ACCESS_CODE=lobe66
部署的时候使用这个参数，首次打开的时候会显示要验证code，如果这个时候不写access code，只填api key，就会直接跳过这个code验证。
请问这个流程本身就是这么设计的吗？ 谢谢！

**Comment by arvinxx** [comment, marked as answer]
是的。密码和api key 2选1
